### PR TITLE
fix(infiniteGrid): deal with -webkit-line-clamp

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#149 

## Details
infiniteGrid couldn't use -webkit-line-clamp property.

## Preferred reviewers
@somebody

### CODE REVIEW

File: assets/css/item.css [hunk 1]
```diff
@@ -19,4 +19,11 @@ body {background-color: #f1f2f6;}
   .item {
     width : 16.66666667%;
   }
+}
+
+p.desc {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp:1;
+  display: -webkit-box;
```
Comment (jongmoon) [inline assets/css/item.css]: Doesn't it need to consider other browser like firefox?

Comment (sculove) [inline assets/css/item.css]: ok I'll also add -moz prefix

Comment (jongmoon) [inline assets/css/item.css]: How about IE?

InfiniteGrid support IE8+ (http://naver.github.io/egjs/latest/doc/eg.InfiniteGrid.html)

Comment (sculove) [inline assets/css/item.css]: IE does not support `line-clamp`.
http://caniuse.com/#feat=css-line-clamp

OK. I will workaround codes

**Comment by jongmoon** [comment]
LGTM!

Maybe we should consider adopting `auto hidden functionality` later.

**Comment by mixed** [comment]
LGTM

**Pull request merged**